zebROS_ws/src/tf_object_detection/src/object_detection.cpp

```diff
@@ -1,176 +0,0 @@
-#include <ros/ros.h>
```

Comment:
I think this is code which converts RGB+depth into to world coordinates, so it needs to be left in the repo

Reply:
This still needs to be restored.